eloquent-mutators.md

```diff
@@ -267,7 +267,9 @@ When defining a `date` or `datetime` cast, you may also specify the date's forma
         'created_at' => 'datetime:Y-m-d',
     ];
 
-When a column is cast as a date, you may set its value to a UNIX timestamp, date string (`Y-m-d`), date-time string, or a `DateTime` / `Carbon` instance. The date's value will be correctly converted and stored in your database:
+> {note} It's important to note that when using a custom date format using the `date` or `datetime` cast that the `app.timezone` will be applied.
```

Comment:
Not sure we have to insist on that point but, in ISO-8601 strings, only the timezone offset of the date will appear without keeping the whole (`app.timezone`) information in the string.

It's correct regarding the datetime but it means guessing what was the exact timezone from it accurately is not possible (multiple timezone names can result into the same offset, and a named timezone can have multiple possible offsets due to DST).